tools/sctool/src/main/java/com/horizen/ScBootstrappingToolCommandProcessor.java

```diff
@@ -342,6 +347,121 @@ private void processGenerateCertProofInfo(JsonNode json) {
         printer.print(res);
     }
 
+    private void processGenerateCertWithKeyRotationProofInfo(JsonNode json) throws Exception {
+        if (!json.has("signersPublicKeys") || !json.get("signersPublicKeys").isArray()) {
+            printGenerateCertProofInfoUsageMsg("wrong signersPublicKeys");
+            return;
+        }
+
+        List<String> signersPublicKeys = new ArrayList<String>();
+
+        Iterator<JsonNode> pksIterator = json.get("signersPublicKeys").elements();
+        while (pksIterator.hasNext()) {
+            JsonNode pkNode = pksIterator.next();
+
+            if (!pkNode.isTextual()) {
+                printGenerateCertProofInfoUsageMsg("wrong signersPublicKeys format");
```

Comment:
Please implement printGenerateCertWithKeyRotationProofInfoUsageMsg() function that prints information about usage generateCertWithKeyRotationProofInfo and call it instead of printGenerateCertProofInfoUsageMsg in this place and all others in processGenerateCertWithKeyRotationProofInfo().

Reply:
Done